Gemfile

```diff
@@ -15,16 +15,16 @@ gem 'sqlite3'
 # Use Puma as the app server
 gem 'puma', '~> 3.7'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
+# gem 'sass-rails', '~> 5.0'
```

Comment:
Why not just delete instead of comment out?

Reply:
Excellent question!   All of this stuff was added by the rails generator, including pinning to versions.  I was feeling tentative, so I commented stuff out.  I would be happy to remove cruft if we prefer -- we can always get it back from git history or another rails app if we need it.  What do you think?

Reply:
My opinion on this one is that we might as well remove this stuff from the Gemfile. These are fairly small changes that would be easy to reinstate, even just by - as you say - running the rails generator again to figure out what we're missing.

I'm less certain about #30 - I believe that removing some of those directories can come back to bite us later and it might be worth just leaving them in there?